static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,32 @@ define([
         }
     }
 
+    function initMostPopularCard(el) {
+        var height;
+        var $adSlot = $(el);
+        var $mostPopTabs;
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
+            fastdom.read(function () {
+                height = $adSlot.dim().height;
+                $mostPopTabs = $('.js-most-popular-footer .tabs__pane');
+            });
+
+            fastdom.write(function () {
+                $mostPopTabs.css('height', height);
+            });
+        }
+
+        createToggle(el);
+    }
+
     CommercialComponent.prototype.postLoadEvents = {
         bestbuy: function (el) {
             new Tabs().init(el);
         },
         capi: createToggle,
         capiSingle: createToggle,
-        paidforCard: createToggle
+        paidforCard: initMostPopularCard
```

Comment:
@regiskuckaertz You wouldn't even need to return the elements, necessarily. Perhaps we could do something like this -

    function join() {
        var actions = Array.apply(null, arguments);
        return function joined() {
            forEach(actions, function (action) {
                action.apply(this, arguments);
            })
        };
    }

and then

    paidForCard : join(createToggle, initMostPopularCard)